src/lib/types/Languages.d.ts

```diff
@@ -138,6 +138,9 @@ declare module 'klasa' {
 			FOOTER: string;
 			TITLE: string;
 		};
+		COMMAND_DISCORD_STAUTS_DECRIPTION: string;
+		COMMAND_DISCORD_STAUTS_EMBED_DECRIPTION: (incident: string) => string;
+		COMMAND_DISCORD_STATUS_EMBED_FOOTER: (time: string) => string;
```

Comment:
See above comment about standard format for embed internationalization

Reply:
Resolved by [24c4825](https://github.com/tuataria/steve/pull/51/commits/24c482586f52b18c33020abb67695e8921815edb)